docs/products/cassandra/howto/enable-cross-cluster-replication.md

```diff
@@ -1,76 +1,63 @@
 ---
 title: Enable cross-cluster replication in Aiven for Apache Cassandra®
+pro: true
```

Comment:
Already raised it on slack, but just for being extra sure! Pro Platform is now Aiven Entreprise